Update multiple URL rewrites (#672)
magento/merchdocs: src/marketing/url-rewrite-update-all.md

---
title: Multiple URL Rewrites
---

To create a single URL rewrite, refer to [Creating URL Rewrites]({% link marketing/url-rewrite-create.md %}).

> To quickly update URL rewrites for multiple or all products simultaneously, follow the steps below.

Reviewer: ```suggestion
You can quickly update URL rewrites for multiple or all products simultaneously using the following steps.
```

> ## Update multiple URL rewrites

Reviewer: You need a blank line here to pass the MD lint check.

1. On the _Admin_ sidebar, go to **Catalog** > **Products**.
1. Select all of the products for which you want to update URL rewrites.
1. Under _Actions_, choose **Update attributes** to update multiple or all rewrites.

> 1. Under _PRODUCTS INFORMATION_, choose the **Websites** tab.

Reviewer: ```suggestion
1. Under _PRODUCTS INFORMATION_, click the **Websites** tab.
```
